src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -42,11 +46,11 @@ public final class Guardrails implements GuardrailsMBean
 {
     public static final String MBEAN_NAME = "org.apache.cassandra.db:type=Guardrails";
 
-    private static final GuardrailsConfigProvider CONFIG_PROVIDER = GuardrailsConfigProvider.instance;
+    public static final GuardrailsConfigProvider CONFIG_PROVIDER = GuardrailsConfigProvider.instance;
     private static final GuardrailsOptions DEFAULT_CONFIG = DatabaseDescriptor.getGuardrailsConfig();
 
     @VisibleForTesting
-    static final Guardrails instance = new Guardrails();
+    public static final Guardrails instance = new Guardrails();
```

Comment:
Nope, I guess I changed it while testing something and forgot to leave it as it was. Reverted.

Reply:
I still see it public? Maybe you didn't commit that change

Reply:
I reverted it to default visibility in the same commit where I addressed most of the review feedback, [this one](https://github.com/apache/cassandra/pull/1546/commits/15099b433a8fb1d543526c5b9f6b0be1cf74e456#diff-ee9b238deb86b4f297d2bd8a400cdfd799e33d08238bde9652b73a2f64a89c08). Maybe you were looking this second time at `Guardrails.CONFIG_PROVIDER`, a couple of lines above, that has also become public because we need it in the disk usage monitor?